src/index.js

```diff
@@ -0,0 +1,84 @@
+const fs = () => window[window._fs_namespace];
+let didInit = false;
+
+const ensureSnippetLoaded = () => {
+  const snippetLoaded = !!fs();
+  if (!snippetLoaded) {
+    throw Error('FullStory is not loaded, please ensure the FullStory snippet is executed before calling FullStory API functions');
+  }
+  return true;
+};
+
+const hasFullStoryWithFunction = (...testNames) => {
+  const functionsCreated = () => testNames.reduce((acc, current) => acc && fs()[current], true);
```

Comment:
Instead of using `reduce`, we can use `every`, which will also short circuit if `false` is returned.

Reply:
good catch!